src/applications/combined-debt-portal/combined/sass/partials/_balance-card.scss

```diff
@@ -0,0 +1,25 @@
+@import "~@department-of-veterans-affairs/formation/sass/shared-variables";
+
+.card-balance {
+  font-family: "Bitter";
+  font-size: 24px;
+}
+
+.card-heading {
+  font-family: "Bitter";
+  font-size: 17px;
```

Comment:
Can we use design system classes instead of writing CSS? `vads-u-font-size--h4`
https://design.va.gov/foundation/utilities/font-size